src/nvim/lua/treesitter.c

```diff
@@ -1549,22 +1549,105 @@ int tslua_parse_query(lua_State *L)
   return 1;
 }
 
-static const char *query_err_string(TSQueryError err)
+static const char *query_err_to_string(TSQueryError error_type, size_t row, size_t column,
+                                       const char *message)
 {
-  switch (err) {
+  const char *msg = "";
```

Comment:
You don't need to initialize with a string here since the `switch` has a default case

```suggestion
  const char *msg;
```